.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,23 +8,26 @@ assignees: ''
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+Give a clear and concise description of what the bug is.
 
 **To Reproduce**
+Zip up a project that reproduces the behavior and attach it by dragging it here.
```

Comment:
Think it's good to let folks know that GitHub supports zipped attachments.

Reply:
oh nice! is that new?

Reply:
Not sure!